### PR TITLE
chore: get command instance from other contract model

### LINKED ISSUE #45: Get command instance from other contract model
## Expected Behavior
- The command already implemented in currency and currency-extension model should be imported from module.
## Actual Behavior
- The command already implemented in currency and currency-extension model has been implemented again.
## Specifications
  - commit; b05af16f753bd4d50a9f08e5a78d3c94c6fcbb0a
  - os; 13.3.1 (22E261)
  - go version; go version go1.19.8 darwin/arm64

### PR DESCRIPTION
**Pull request by soonkuk**
### Pull-request Type (fix, feat, bug, doc, chore, test, etc...) 
- chore
### Current Behavior (Link to an open issue)
- close #45 
### New Behavior (if this is a feature change)
- some command imported from mitum-currency and mitum-currency-extension module

### CODE REVIEW

File: cmds/operation.go [hunk 1]
```diff
@@ -1,42 +1,47 @@
...
+	TransferSecurityTokensPartition TransferSecurityTokensPartitionCommand             `cmd:"transfer-security-token" help:"transfer security tokens by partition"`
+	RedeemTokens                    RedeemTokensCommand                                `cmd:"redeem-tokens" help:"redeem tokens from tokenholder"`
+	AuthorizeOperators              AuthorizeOperatorsCommand                          `cmd:"" name:"authorize-operator" help:"authorize operator"`
+	RevokeOperators                 RevokeOperatorsCommand                             `cmd:"" name:"revoke-operator" help:"revoke operator"`
+	SetDocuments                    SetDocumentsCommand                                `cmd:"" name:"set-documents" help:"set sto documents"`
+	CurrencyRegister                currencycmds.CurrencyRegisterCommand               `cmd:"" name:"currency-register" help:"register new currency"`
+	CurrencyPolicyUpdater           currencycmds.CurrencyPolicyUpdaterCommand          `cmd:"" name:"currency-policy-updater" help:"update currency policy"`
+	SuffrageInflation               SuffrageInflationCommand                           `cmd:"" name:"suffrage-inflation" help:"suffrage inflation operation"`
```
Comment (wyuinche) [inline cmds/operation.go:23]: It seems suffrage-inflation should be replaced, too.
And will isaac cmds such as suffrage-candidate(... etc) not be replaced?

Comment (soonkuk) [reply cmds/operation.go:23]: @wyuinche Ok I understand it. I will also replace the isaac commands, too.

Comment (soonkuk) [reply cmds/operation.go]: @wyuinche I added it.

Comment (soonkuk) [reply cmds/operation.go]: @wyuinche When I run the compiled program it returns error. So I will fix it.

**Review by wyuinche — APPROVED** [review]

**Comment by soonkuk** [comment]
@wyuinche I have finished this.

**Review by wyuinche — APPROVED** [review]
LGMT!

**Pull request merged**